gem/client/css/variables/_variables.scss

```diff
@@ -35,5 +35,14 @@ $colorWhiteSmoke: #ececec;
 $colorPurple: #7300FF;
 $colorYellow: #FFFC80;
 $colorTurq: #A4EED2;
-$colorOrange: #FCAB88; 
-$colorPink: #FF87D9; 
+$colorOrange: #FCAB88;
```

Comment:
@miltontony absolutely, I'm sort of patching a prototype that has to work now and overall GEM has issue we should rather fix - will chat to Johan